src/ffi/dial_ffi.rs

```diff
@@ -241,7 +252,7 @@ pub unsafe extern "C" fn dial(
     }) {
         Ok(s) => s,
         Err(e) => {
-            log::error!("Error building GRPC proxy reason : {e:?}");
+            log::error!("Error building GRPC proxy reason : {}", e);
```

Comment:
More informative error on timeout. Displays "deadline has elapsed" in the error instead of "Elapsed(())". Should be safe as all `Boxed(dyn Error)` have Display implemented.